x-pack/plugins/observability_ai_assistant/server/functions/recall.ts

```diff
@@ -250,18 +229,27 @@ async function scoreSuggestions({
     (
       await client.chat('score_suggestions', {
         connectorId,
-        messages: [extendedSystemMessage, newUserMessage],
+        messages: [...messages.slice(-1), newUserMessage],
```

Comment:
nit: Can we use `last` instead?

```suggestion
        messages: [last(messages), newUserMessage],
```